loadbalancer/build.gradle

```
@@ -0,0 +1,25 @@
+plugins {
+	id 'org.springframework.boot' version '2.4.5'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+}
+
+group = 'com.io'
```

```suggestion
group = 'com.io.loadbalancer'
```
Use the same convention as in imagemanager app